packages/cozy-stack-client/src/KonnectorCollection.js

```diff
@@ -17,8 +20,44 @@ class KonnectorCollection extends AppCollection {
     throw new Error('destroy() method is not available for konnectors')
   }
 
-  async update() {
-    throw new Error('update() method is not available for konnectors')
+  async launch(slug) {
```

Comment:
An optional accountId (like the cli) or triggerId parameter would be useful when there are more than 1 trigger for the given slug

Reply:
Done, thanks. options.accountId can be passed to KonnectorCollection::launch. For a triggerId, the TriggerCollection::launch can be used.